src/components/EditorHeader/components/MetadataButton/MetadataButton.js

```diff
@@ -38,18 +38,17 @@ const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
 
 function MetadataButton({
   disabled,
-  goldStandard,
+  goldStandardCount,
   id,
-  lines,
   metadata,
-  pages,
-  score,
-  status,
-  transcribers
+  transcriberCount,
+  transcription
 }) {
   const targetEl = React.useRef(null)
   const [isOpen, toggleDrop] = React.useState(false)
 
+  const { pages, status, transcribed_lines } = transcription
```

Comment:
1. can `transcription` be `null`? If yes, you might want to make this `... = transcription || {}` or something. (Not sure if null will auto-overwrite the values in defaultProps, mind you.)

2. I assume if you ever have `transcription`, that object will (since it's coming from the API) always have pages, status, transcribed_lines... and even if it didn't, the worst that can happen is you get status = undefined. OK. 👌

Reply:
Good call. Def won't hurt to add a fallback value here